src/Summoner/Template.hs

```diff
@@ -326,31 +332,27 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
             "http://stackage.org/package/" <> repo <> "/badge/nightly"
         stackLinkNightly :: Text =
             "http://stackage.org/nightly/package/" <> repo
-
-        stackBadges :: Text = memptyIfFalse stack
-            [text|
-            [![Stackage Lts](${stackShieldLts})](${stackLinkLts})
-            [![Stackage Nightly](${stackShieldNightly})](${stackLinkNightly})
-            |]
-
+        stackLtsBadge :: Text =
+            makeBadge "Stackage Lts" stackShieldLts stackLinkLts
+        stackNightlyBadge :: Text =
+            makeBadge "Stackage Nightly" stackShieldNightly stackLinkNightly
+            
         travisShield :: Text =
-          "https://secure.travis-ci.org/" <> owner <> "/" <> repo <> ".svg"
+            "https://secure.travis-ci.org/" <> owner <> "/" <> repo <> ".svg"
         travisLink :: Text =
-          "https://travis-ci.org/" <> owner <> "/" <> repo
-        travisBadge :: Text = memptyIfFalse travis
-            [text|[![Build status](${travisShield})](${travisLink})|]
+            "https://travis-ci.org/" <> owner <> "/" <> repo
+        travisBadge :: Text = 
+            makeBadge "Build status" travisShield travisLink
 
         appVeyorShield :: Text =
-          "https://ci.appveyor.com/api/projects/status/github/" <> owner <> "/" <> repo <> "?branch=master&svg=true"
+            "https://ci.appveyor.com/api/projects/status/github/" <> owner <> "/" <> repo <> "?branch=master&svg=true"
         appVeyorLink :: Text =
-          "https://ci.appveyor.com/project/" <> owner <> "/" <> repo
-        appVeyorBadge :: Text = memptyIfFalse appVey
-            [text|[![Windows build status](${appVeyorShield})](${appVeyorLink})|]
+            "https://ci.appveyor.com/project/" <> owner <> "/" <> repo
+        appVeyorBadge :: Text = 
+            makeBadge "Windows build status" appVeyorShield appVeyorLink
 
-        licenseShield :: Text =
-          "https://img.shields.io/badge/license-" <> T.replace "-" "--" license <> "-blue.svg"
-        licenseLink :: Text =
-          "https://github.com/" <> owner <> "/" <> repo <> "/blob/master/LICENSE"
+        makeBadge :: Text -> Text -> Text -> Text
+        makeBadge title shield link = [text|[![$title]($shield)]($link)|]
```

Comment:
I've just discovered that `neat-interpolation` adds newlines. So I see the following output:

```
# foo

[![Hackage](https://img.shields.io/hackage/v/foo.svg)](https://hackage.haskell.org/package/foo)

[![MPL-2.0 license](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](LICENSE)

[![Stackage Lts](http://stackage.org/package/foo/badge/lts)](http://stackage.org/lts/package/foo)

[![Stackage Nightly](http://stackage.org/package/foo/badge/nightly)](http://stackage.org/nightly/package/foo)
```

Looks like we can't use `neat-interpolation` for `makeBadge` function and we should return to previous version with `<>`.